modules/ztp-image-based-upgrade-generate-seed-image.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image. The image generation does the following:
+
+* Prepares for precaching operations
+* Stops the `kubelet` service, running containers, and the CRI-O runtime on the managed cluster
+* Backs up platform configurations and artifacts
+* Creates and pushes the OCI images
```

Comment:
Consider adding a note here that it's expected to lose API availability while LCAO generates the seed image